style.css

```diff
@@ -10,9 +9,29 @@
   font-family:Arial;
   font-size:17px;
   padding:16px 31px;
+  text-align:center;
   text-decoration:none;
   text-shadow:0px 1px 0px #2f6627;
+  width: 80px;
+  max-width: 80px;
+}
+
+.capture {
+  background-color:#0f9d58;
+}
+
+.capture:hover {
+  background-color:#34a853;
+}
+
+.serialize {
```

Comment:
I didn't add hover, because it's not clickable when it's serializing.  I figured that changing the color on hover was to indicate that the element had a click event.